gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,6 @@
+#Thu Sep 26 17:03:44 SGT 2019
```

Comment:
I think we don't need to change this file.

Reply:
Should I change it back? It was suggested by IntelliJ, but I'm not sure why.